lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -417,6 +417,13 @@ bool CPUBackend::isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const {
   return true;
 }
 
+bool CPUBackend::shouldLower(Node *N) const {
+  if (N->getKind() == Kinded::Kind::ConvolutionNodeKind)
+    return false;
+  else
```

Comment:
nit: can remove `else` here
can even do `return (N->getKind() != Kinded::Kind::ConvolutionNodeKind);`, although it's a bit fragile, because next person changing this function will likely go back to:
```
if (something1)
  return false;
if (something2)
  return false;
return true;
```